src/test/java/de/hs_mannheim/informatik/ct/end_to_end/RoomControllerTest.java

```diff
@@ -177,10 +182,12 @@ public void checkInFilledRoom() throws Exception {
                         .with(csrf()))
                 .andExpect(status().isOk());
     }
-
+    
+    // TODO create a test for a full room, the proposed one here seems tests functionality we do not have (yet?)
+    // It requires a room full notification even before the check-in is attempted (which would make sense, of course)
+    /*
```

Comment:
@maxones25: I don't understand this test? Seems to test something we don't have? Or did I break something during my changes?

Reply:
@oliverhummel I believe you removed a code block in the checkIn method in the RoomController. Previously, when a room was full and overwriting the room was not allowed, a room-full page was displayed. The commented out test checked this case. I have added the code back. Since we currently allow overriding the room, the code will be ignored in the production system and therefore should not cause any problems.